sdks/java/core/src/main/java/org/apache/beam/sdk/util/Credentials.java

```diff
@@ -137,13 +138,27 @@ public static Credential getCredential(GcpOptions options)
   private static Credential getCredentialFromFile(
       String keyFile, String accountId, Collection<String> scopes)
```

Comment:
accountId is @Nullable depending on the type of key (keyFile is not in the getCredentialFromFile)